internal/runtime/runtime_test.go

```diff
@@ -328,13 +271,8 @@ func TestProcessMessageJSON(t *testing.T) {
 	runtime.SetTransforms([]appcontext.AppFunction{transform1})
 
 	result := runtime.ProcessMessage(context, envelope)
-	if !assert.Nil(t, result, "result should be null") {
-		t.Fatal()
-	}
-
-	if !assert.True(t, transform1WasCalled, "transform1 should have been called") {
-		t.Fatal()
-	}
+	assert.Nil(t, result, "result should be null")
```

Comment:
why not require for these?

Reply:
Because the remaining validation can still executed w/o crashing.